net/DelaySocket.cpp

```diff
@@ -23,9 +23,16 @@ std::once_flag Delay::DelayPollOnceFlag;
 /// Reads from fd, delays that and then writes to _dest.
 class DelaySocket : public Socket {
     int _delayMs;
-    enum State { ReadWrite,      // normal socket
-                 EofFlushWrites, // finish up writes and close
-                 Closed };
+    enum class State : uint8_t { ReadWrite,      // normal socket
+                                 EofFlushWrites, // finish up writes and close
+                                 Closed };
+
+    static const char* toString(State s) noexcept {
```

Comment:
I had the same thought, but then didn't want to change even more code. So yes, we could reuse our macro .. even if it would render the enum underling bigger (I used uint8_t .. the macro uses int in general, no biggy).

Reply:
Resolved with revised commit `Use scoped enums with efficient types (STATE_ENUM)`, missed it.